src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+public class Tagging {
+	private ArrayList<String> addTag = new ArrayList<String>();
+	private ArrayList<String> deleteTag = new ArrayList<String>();
```

Comment:
Recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
Please check and differentiate the 2 diagrams in the activity description for better understanding.